### PR TITLE
It's the correct code in c++

### PR DESCRIPTION
**Pull request by meghasharma123**
Please Check my code and add it to yours . Thank you

### CODE REVIEW

File: factorial_cpp.cpp [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+	    cin>>num;
+	    if(num==0||num==1){
+	      cout<<1<<endl;
+	    }
+	    else if(num>1){
+	         for(int i=1;i<=num;i++){
+	               fact *= i;
+	           }
+	          cout<<fact<<endl;
+	          fact = 1;
```
Comment (coderbond007) [inline factorial_cpp.cpp]: Why not a local variable within the scope of while loop of test case.
`
while(t--){
int fact=1
`

Nitpick: Change it to long long

**Review by coderbond007 — COMMENTED** [review]
Add the problem link in comment after the header section.

File: factorial_cpp.cpp (hunk 1 above)
Comment (meghasharma123) [reply factorial_cpp.cpp]: I don't think it much affect the code right? let me correct it once too for you..Is that ok?

**Review by meghasharma123 — COMMENTED** [review]
done

**Comment by coderbond007** [comment]
LGTM, merging it to master.

**Comment by meghasharma123** [comment]
Thank you...

**Pull request merged**